chunk/chunk.go

```diff
@@ -38,17 +38,20 @@ var (
 type Chunk interface {
 	Address() Address
 	Data() []byte
+	PinCounter() uint64
 }
 
 type chunk struct {
-	addr  Address
-	sdata []byte
+	addr       Address
+	sdata      []byte
+	pinCounter uint64
 }
 
-func NewChunk(addr Address, data []byte) Chunk {
+func NewChunk(addr Address, data []byte, pinCounter uint64) Chunk {
```

Comment:
please use `withPinCounter` the same way as pushsync uses tags: https://github.com/ethersphere/swarm/pull/1392/files#diff-e179054c3d912cc0aaa6f11efb991521R58
then at least 5 files wont be modified

Reply:
Done